openbrush.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
```

Comment:
what is the purpose for this file?

Reply:
sorry, this file should not have been in this pull request, will remove.
moving this file to:
https://gitlab.com/panjea.developers/openbrushqt

This fork will use pyqt for UI, gamepads for input, and interface with blender grease pencil. `openbrush.py` is the frontend that builds if required and then runs.

Reply:
interesting! There's a PR, currently in progress, for some rather significant monoscopic mode changes that you might want to follow: https://github.com/icosa-gallery/open-brush/pull/136